readme.md

```diff
@@ -1,79 +1,52 @@
 
 # Welcome to the Stalin Sort Repo :clipboard:
 
-Stalin Sort in any language you like! What is Stalin Sort?
+## What is Stalin Sort?
 
-![poster](stalin-sort.jpeg)
```

Comment:
i like the simplicity of this tweet as one of the first things because it summarizes better than anything what this repo is all about

Reply:
what do you think?

Reply:
Maybe it can be placed at the top, I'll change and see how it goes